editor/editor_property_name_processor.cpp

```diff
@@ -91,7 +91,27 @@ String EditorPropertyNameProcessor::_capitalize_name(const String &p_name) const
 	return capitalized;
 }
 
-String EditorPropertyNameProcessor::process_name(const String &p_name, Style p_style) const {
+StringName EditorPropertyNameProcessor::_get_context(const String &p_name, const String &p_property, const StringName &p_class) const {
+	if (p_property.is_empty() && !p_class) {
```

Comment:
```suggestion
	if (p_property.is_empty() && p_class == StringName()) {
```
There's no `operator bool()` for `StringName` so unsafe to do, unsure what operation even is carried out here to achieve this

Reply:
It uses `operator const void *()` so it's safe. But yeah, it takes more steps when the StringName is not empty. I'll change this.